templates/partial/search-no-results.html.twig

```diff
@@ -0,0 +1,2 @@
+{% set index = route|replace({'search':'index'}) %}
+<p>No results found. Try a simpler search or <a href="{{path(index)}}">return to the document list</a>.</p>
```

Comment:
I forsee a comment from the PI like "We're not always going back to a _document_ list."

Reply:
I've changed this now to "or browse all items". If that doesn't end up working, we can always add a proper label for that if necessary.